src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -631,6 +631,14 @@ $pf-c-dropdown--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "
   background: var(--pf-c-dropdown__menu--BackgroundColor);
   background-clip: padding-box;
   box-shadow: var(--pf-c-dropdown__menu--BoxShadow);
+
+  &:only-child,
+  &.pf-m-static {
+    position: static;
+    top: auto;
+    z-index: auto;
+    min-width: auto;
```

Comment:
Tested these styles out in React locally by replacing the current inline style that uses `position: revert; min-width: min-content` on `pf-c-dropdown__menu`, and they do work when `popperMatchesTriggerWidth={false}` is passed into Popper. Doing that may result in the following, though:

![Dropdown toggle wider than content](https://user-images.githubusercontent.com/70952936/182437459-d5d4afa8-ac46-4cd8-9b1a-fceb47c9ff6f.png)

Changing `min-width` to `min-content` from these styles (without needing `popperMatchesTriggerWidth={false}`) looks to more closely match how the "inline" dropdown looks. Would `auto` be better to use since this is a modifier class that could be used outside of just these dropdown-like components?

Reply:
Just wanted to mention that another solution we discussed was if popper could use `min-width` instead of `width` with `poppermatchesTriggerWidth` since we don't always want the width to match (I think that would be updated here - https://github.com/patternfly/patternfly-react/blob/8bcbea20cbe53d6d5ba472a0a94b40860ff5e15c/packages/react-core/src/helpers/Popper/Popper.tsx#L252-L267), but I don't see a downside to use `min-width: min-content` on the static menu, either. Since dropdown item text doesn't wrap, this will fix other conditions where there is overflow in the menu from long menu items.